app/api/actions/resources.js

```diff
@@ -23,7 +21,7 @@ function fetchResource(id, params = {}) {
 function fetchResources(params = {}) {
   return createApiAction({
     endpoint: 'resource',
-    params: Object.assign({}, params, { pageSize: 100 }),
+    params: { ...getParamsWithTimes(params), ...{ pageSize: 100 } },
```

Comment:
- [ ] The ... is not required for the second one. `{ ...getParamsWithTimes(params), pageSize: 100 }`.